tools/please_go/install/install.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/thought-machine/please/tools/please_go_embed/embed"
 )
 
-const baseWorkDir = "_build"
+const baseWorkDir = "_work"
```

Comment:
No, it's to better differentiate between plz's `_build` and go's `wordir`, as it can get confusing when the path that you are reading (or is shown to the terminal) contains the same token.